src/main/java/sirius/biz/importer/BaseImportHandler.java

```diff
@@ -582,6 +641,22 @@ public Object renderExportRepresentation(Object entityId) {
                                        exportRepresentationMapping);
     }
 
+    /**
+     * Builds a standard message that the given entity cannot be saved.
+     * <p>
+     * This message is mostly used with {@link sirius.biz.process.ErrorContext} methods which accept an extra failure description.
+     * For instance, when an entities required field is not populated, a standard message like this will be logged:
+     * {@code Field 'MyField' is required.}
+     * As to would be rather nice to tell the end user that an entity couldn't be saved due to the error.
+     * {@code MyEntity cannot be saved. Field 'MyField' is required.}
+     *
+     * @param entity the entity to extract its label
+     * @return the formatted message
+     */
+    public String getCannotSaveMessage(E entity) {
```

Comment:
build_ oder create_ (optional) ?

+ add a reference to the method where this can be used using @see or the like (?)